CHANGELOG.md

```diff
@@ -18,6 +18,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   left after children were wrapped. [#930]
 - FSDP: Add support for saving optimizer state when using expert replicas with FSDP.
 
+### Fixed
+- FSDP: fixing handling of internal states with state_dict and load_state_dict.
+  This is to support optimizer state handling when some of the parameters are
```

Comment:
This line can be deleted I think with the above modification.

Reply:
Updated.